lighthouse-core/audits/is-on-https.js

```diff
@@ -72,6 +72,11 @@ class HTTPS extends Audit {
       extendedInfo: {
         formatter: Formatter.SUPPORTED_FORMATS.URL_LIST,
         value: insecureRecords
+      },
+      details: {
```

Comment:
two thoughts on details:

1) I get the point that the `type: 'text'` on every node is basically JSON HTML, but it seems like it's valuing simplicity in the report renderer at the expense of more verbose and less readable JSON results. A simple fix in this case would be to let the header type determine the type of rendering used for items (and if we ever encounter the need for mixed-type items we can deal with it then)

2) This can wait for a followup PR: we shouldn't have an `extendedInfo` and a `details` as those are basically synonyms. Either we should pick clearer names for the two (one is for report rendering and the other is going to be for information someone somewhere might need maybe), or just have `details` and have `extendedInfo` be a property on it